### ISSUE TITLE
Improve the Limit/Extend syntax or provide a GUI mechanism.

### ISSUE THREAD

**Issue by wkthompson123** [ABCD-STUDY/DEAP#58]
Self expanatory

**Comment by HaukeBartsch** [comment]
Yes, that is a good request for the future. There are many statistical packages that do this using a GUI instead of using scripting. Currently we only support scripting because that is the way that would allow an expert to implement any type of Limit/Extend set.

With more support for development we would be able to mimic the GUI driven approach provided by other commercial software vendors.

**Comment by klkerr** [comment]
Would it be possible to add a window showing descriptive data, such as the values possible for that variable, when a variable is entered? It would be helpful to have something like this rather than going back and forth to "Explore" to see what possible values the data take on, especially for categorical variables. The variable "demo_comb_income_v2b" is a good example of this; there's no way I'm going to memorize the different levels of this variable. Also, if it could start to fill in the variable name as you type, that would be great, too.
